react/fields/Autosuggest/Autosuggest.less

```diff
@@ -0,0 +1,56 @@
+@import (reference) "~seek-style-guide/theme";
+
+.container {
+  position: relative;
+  display: inline-block;
+  width: 100%;
+}
+
+.suggestionsContainer {
+  display: none;
+
+  .containerOpen & {
+    z-index: 1;
+    display: block;
+    position: absolute;
+    background-color: @sk-white;
+    box-shadow: 0 9px 30px rgba(0, 0, 0, 0.4);
+    padding: @grid-row-height 0;
+    border-radius: 2px;
+    min-width: 100%;
+    max-width: 100%;
+    max-height: 385px;
+    overflow-y: auto;
+    top: @grid-row-height * 6;
+
+    @media only screen and (min-width: 920px) {
+      max-width: initial;
```

Comment:
let's start with removing the media query and we'll deal with exceptions as they arise.